frontend/src/routes/Root/index.jsx

```diff
@@ -30,9 +30,18 @@ const queryClient = new QueryClient();
 function Root(){
     const theme = useThemeStore((state) =>state.theme)
     const logged = useUserStore((state) => state.logged)
+    const navigate = useNavigate();
 
     const location = useLocation();
 
+    useEffect(()=>{
+      if (!logged){
+        navigate("/login");
+      }else if (location.pathname === "/"){
+        navigate("/dashboard")
+      }
+    },[logged])
+
```

Comment:
if the role isn't admin, what happens then?

Reply:
heheha

Reply:
pagina errada